uint64 timestamp
waterlinked_a50/TransducerReport report